jdaviz/configs/cubeviz/plugins/spectral_extraction/tests/test_spectral_extraction.py

```diff
@@ -575,7 +576,6 @@ def test_spectral_extraction_unit_conv_one_spec(
     assert spectrum_viewer.state.y_display_unit == "MJy"
 
 
-@pytest.mark.usefixtures('_jail')
```

Comment:
as added in the note in `cache-download.yml`, jail is not compatible with retrieving from the cache (unless we are smarter in `cached_url` to detect the non-jailed path or create symlinks, copy data, etc.  I don't see any reason why this test needs to be sandboxed, so I'm just removing it here.  If we ever have the need for both, we can build the infrastructure needed for that then.